### PR TITLE
Use city as centre for TokenSelector, instead of the centre of the tile

### PR DESCRIPTION
**Pull request by daniel-sousa-me**
I think this only affects Pile o' Bones in 1882, but looks nicer

### CODE REVIEW

**Comment by tobymao** [comment]
show a picture

**Comment by jenf** [comment]
also does this have any issue with tiles on the edges of the board?

**Comment by daniel-sousa-me** [comment]
It is not checking for the edges, but this commit doesn't make that any better or any worse. I thought about also addressing that, but in the end it's very hard to imagine a case where that would be relevant, since the tokens never appear very far from the hex

**Comment by daniel-sousa-me** [comment]
Old:

image

New:

image

File: assets/app/view/game/part/city_slot.rb [hunk 1]
```diff
@@ -22,6 +22,7 @@ class CitySlot < Base
         needs :tile_selector, default: nil, store: true
         needs :reservation, default: nil
         needs :game, default: nil, store: true
+        needs :city_render_location, default: nil
```
Comment (tobymao) [inline assets/app/view/game/part/city_slot.rb:25]: how can the default be nil if on line 114 you expect it to always exist

**Review by tobymao — APPROVED** [review]

**Pull request merged**